Gemfile

```diff
@@ -64,7 +37,7 @@ gem "omniauth-shibboleth"
 gem "clamav"
 
 # for migration reading the license file
-gem "pdf-reader"
+gem "pdf-reader", require: false
```

Comment:
Only needed in a rake task (which already has it required in that task file), shouldn't be loaded in rails environment